### ISSUE TITLE
Long connection times when forwarding RDP connections through SSH

### ISSUE THREAD

**Issue by Nativedude** [FreeRDP/Remmina#452]
Hey guys,
Got a bug on my system with the latest packages through the ppa where any RDP connection that I forward through SSH takes upwards of 2 mins.
I don't see any errors in the debug window and this worked just fine up until I updated today.
I had encountered this about a week or two ago when I was using a build following the Wiki, bypassed it by removing that and just adding the PPA but I guess that's now caught up to the point where this bug was introduced.

Currently running Mint 17.1 with openssh (1:6.6p1-2ubuntu2), remmina_1.1.1-1+648+next+201502081946, & remmina-plugin-rdp_1.1.1-1+648+next+201502081946

I'm sure yall will need some more info so just ask and I'll provide.

Thanks,
Adam D
##

**Comment by Nativedude** [comment]
Hey guys,
Got an update for ya.
Did some more troubleshooting today and found out that it might be an issue with RDP.
Downloaded and compiled v1.2.0-rcgit.2 via the releases tab and tried that out with the same issue.
Honestly not 100% sure why it was working on previous builds and not now but didn't want to keep tracing back through builds to see when it was introduced.
Anywho, so I ran remmina from a terminal window and noticed this error when I tried any of my SSH tunnled connections
[10:43:41:360] [20708:1029912320] [INFO][com.freerdp.client.common.cmdline] - loading channel cliprdr
[10:43:42:382] [20708:1029912320] [ERROR][com.freerdp.core.nego] - Error: SSL_NOT_ALLOWED_BY_SERVER

So I then tried messing with the RDP settings some, I found that I would get that error when security was set to Negotiate, NLA, or TLS.

HOWEVER, it did finally work on RDP.
So I'm thinking this is probably how it always connected and now for some reason it's hanging up on negotiating somewhere.
Keep in mind it does work eventually.
I think maybe a timeout setting or something in the Negotiation is set way to high.

But at least it works again if I set it directly to RDP :)
